examples/rules.mk

```diff
@@ -200,27 +238,21 @@ styleclean: $(STYLECHECKFILES:=.styleclean)
 
 ifeq ($(STLINK_PORT),)
 ifeq ($(BMP_PORT),)
-ifeq ($(OOCD_SERIAL),)
-%.flash: %.hex
+ifeq ($(OOCD_FILE),)
+%.flash: %.elf
```

Comment:
Don't the STLink windows tools expect to see .HEX files? ELF is great for many things but the vendors seem to look for good old .hex files.